### PR TITLE
adds initial classes for vertex and edge

### PR DESCRIPTION
**Pull request by Brett-Constantinoff**
* adds classes for Vertex and Edge
* successfully able to connect two vertices with an edge

### CODE REVIEW

**Review by maxsg5 — COMMENTED** [review]
This is a test comment @Brett-Constantinoff do you get notified if I leave a comment?

**Comment by Brett-Constantinoff** [comment]
Yes, i do

**Comment by Brett-Constantinoff** [comment]
you can also leave individual comments on the code itself

File: src/App.js [hunk 1]
```diff
@@ -60,10 +63,27 @@ export default class App
...
         document.body.appendChild(this.stats.dom);
         
+        this.camera.position.z = 10;
         this.scene.add(this.camera);
 
         this.renderer.setPixelRatio(window.devicePixelRatio);
         this.renderer.setSize(this.sizes.width, this.sizes.height);
+        this.renderer.setClearColor("Grey", 1.0);
+
+        // white light
```
Comment (maxsg5) [inline src/App.js:73]: Light also added to scene.

**Comment by maxsg5** [comment]
LGTM! 
image

**Pull request merged**